src/main/scala/org/hammerlab/guacamole/reads/Read.scala

```diff
@@ -173,9 +220,6 @@ object Read extends Logging {
                     referenceGenome: Option[ReferenceGenome]): Read = {
 
     val isMapped = (
-      // NOTE(ryan): this flag should maybe be the main determinant of the mapped-ness of this SAM record. SAM spec
-      // (http://samtools.github.io/hts-specs/SAMv1.pdf) says: "Bit 0x4 is the only reliable place to tell whether the
-      // read is unmapped."
```

Comment:
The comment is correct, but to be extremely nit picky it's not directly relevant to the code. The SAM spec is saying that if that flag says the read is unmapped, we can't assume anything about the values like alignment start, cigar, etc. -- they might well have non-empty values, and we shouldn't use those values to decide that the read is actually mapped even though that flag says it's unmapped. If the flag says the read is mapped, however, it's totally reasonable for us to also require that those other values are meaningful / non-empty, which is what we do here. (Arguably we could throw an error if not, instead of calling the read unmapped.)